### PR TITLE
Dapla 775 output format

### PR DESCRIPTION
**Pull request by RupinderKaurSSB**
Added new parameter "outputType" to specify the encrypted file format. 
Ex: 
--sourceType=PARQUET
--outputType=PARQUET
This will create encrypted file in .parquet format. If --outputType parameter is not passd while running the pipeline, it will be same as sourceType.

### CODE REVIEW

File: src/main/java/com/google/cloud/solutions/autotokenize/pipeline/EncryptionPipeline.java [hunk 1]
```diff
@@ -153,14 +155,33 @@ public PipelineResult run() throws Exception {
...
+                          .withCodec(CodecFactory.snappyCodec()));
+          break;
+        case "PARQUET":
+          encryptedRecords.apply(
+                  "WritePARQUET",
+                  FileIO.<GenericRecord>write()
+                          .withSuffix(".parquet")
+                          .via(ParquetIO.sink(buildEncryptedSchema()))
+                          .to(cleanDirectoryString(options.getOutputDirectory()) + "/data"));
+          break;
```
Comment (bjornandre) [inline src/main/java/com/google/cloud/solutions/autotokenize/pipeline/EncryptionPipeline.java:183]: What about CSV?

File: src/main/java/com/google/cloud/solutions/autotokenize/pipeline/EncryptionPipeline.java [hunk 2]
```diff
@@ -153,14 +155,33 @@ public PipelineResult run() throws Exception {
...
+                          .withSuffix(".avro")
+                          .to(cleanDirectoryString(options.getOutputDirectory()) + "/data")
+                          .withCodec(CodecFactory.snappyCodec()));
+          break;
+        case "PARQUET":
+          encryptedRecords.apply(
+                  "WritePARQUET",
+                  FileIO.<GenericRecord>write()
+                          .withSuffix(".parquet")
+                          .via(ParquetIO.sink(buildEncryptedSchema()))
```
Comment (bjornandre) [inline src/main/java/com/google/cloud/solutions/autotokenize/pipeline/EncryptionPipeline.java:181]: `buildEncryptedSchema` has already been called, so you can re-use `encryptedSchema`

File: src/main/java/com/google/cloud/solutions/autotokenize/pipeline/EncryptionPipeline.java (hunk 1 above)
Comment (RupinderKaurSSB) [reply src/main/java/com/google/cloud/solutions/autotokenize/pipeline/EncryptionPipeline.java:183]: CsvTokenizationAndOrderingPipeline itself read CSV files, tokenize using DLP and output as CSV. Do we need it?

**Review by bjornandre — APPROVED** [review]
LGTM

**Pull request merged**